### ISSUE TITLE
Downstream task implementation

### ISSUE THREAD

**Issue by iamliuquan** [Xiyue-Wang/TransPath#5]
Hi Xiyue,

Thank you for your excellent work. I really enjoy your paper. 
I am trying to reproduce your results in your paper. Could you provide the details for the finetuning strategy of downstream task？
（eg. classifier, finetune parameter, optimizer and so on）

Many thanks,
Quan Liu

**Comment by Xiyue-Wang** [comment]
Dear Quan Liu,
         Thank you very much for your attention to our work, detailed details and more extended downstream tasks, we are invited, the paper is being submitted for extended submission, please wait patiently, the detailed and extended paper is being prepared, it will be completed soon, we will make it all public , there will be a lot of comprehensive downstream task verification, you will not be disappointed.
Thanks
Xiyue Wang

**Comment by iamliuquan** [comment]
Yes, thank you for your reply Xiyue. One quick question: for the downstream task, will you only finetune the last several layers (classifier) or finetune the whole network including backbone? 
Thanks again for the general sharing.
